analytics/views.py

```diff
@@ -1132,6 +1133,12 @@ def support(request: HttpRequest) -> HttpResponse:
             attach_discount_to_realm(realm, new_discount)
             msg = f"Discount of {realm.string_id} changed to {new_discount} from {current_discount} "
             context["message"] = msg
+        elif request.POST.get("change_subdomain", None) is not None:
+            subdomain = str(request.POST.get("change_subdomain"))
+            do_change_realm_subdomain(realm, new_subdomain=subdomain)
+            realm = Realm.objects.get(id=realm_id)
```

Comment:
We already get the value of `realm`  before the `if`. This is redundant.

Reply:
Yes! this was part of debug that i was trying and forgot to remove this line..